simul/master/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	"github.com/ConsenSys/handel/simul/lib"
+)
+
+var nbOfNodes = flag.Int("nbOfNodes", 0, "number of slave nodes")
+var masterAddr = flag.String("masterAddr", "", "master address")
+
+func main() {
+	flag.Parse()
+	master := lib.NewSyncMaster(*masterAddr, *nbOfNodes)
+	fmt.Println("Master: listen on", *masterAddr)
+
+	select {
+	case <-master.WaitAll():
+		fmt.Printf("[+] Master full synchronization done.\n")
+		master.Reset()
+
+	case <-time.After(3 * time.Minute):
```

Comment:
as discussed in the call - it's best if we can put this timeout as a configurable option

Reply:
Done